common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -7,6 +7,127 @@
 from base_studio_test import StudioCourseTest
 
 from ...pages.studio.settings_advanced import AdvancedSettingsPage
+from ...pages.studio.settings_group_configurations import GroupConfigurationsPage
+
+
+@attr('shard_1')
+class ContentGroupConfigurationTest(StudioCourseTest):
+    """
+    Tests for content groups in the Group Configurations Page.
+    There are tests for the experiment groups in test_studio_split_test.
+    """
+    def setUp(self):
+        super(ContentGroupConfigurationTest, self).setUp()
+        self.group_configurations_page = GroupConfigurationsPage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+    def create_and_verify_content_group(self, name, existing_groups):
+        """
+        Creates a new content group and verifies that it was properly created.
+        """
+        self.assertEqual(existing_groups, len(self.group_configurations_page.content_groups))
+        if existing_groups == 0:
+            self.group_configurations_page.create_first_content_group()
+        else:
+            self.group_configurations_page.add_content_group()
+        config = self.group_configurations_page.content_groups[existing_groups]
+        config.name = name
+        # Save the content group
+        self.assertEqual(config.get_text('.action-primary'), "Create")
+        self.assertTrue(config.delete_button_is_absent)
+        config.save()
+        self.assertIn(name, config.name)
+        return config
+
+    def test_no_content_groups_by_default(self):
+        """
+        Scenario: Ensure that message telling me to create a new content group is
+        shown when no content groups exist.
```

Comment:
I'd indent this follow on line so that the scenario stands out from the other clauses